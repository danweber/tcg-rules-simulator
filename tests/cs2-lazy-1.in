VERSION:2
FIELD1:Victim,CS1-02
FIELD2:
HAND2:CS2-Lazy
GAME:P2 T1 MAIN 9
DUMP

ECHO: -- play, get suspended immediately
CMD:PLAY CS2-Lazy Lazy
DUMP

ECHO: -- try to attack, fail
CMD:ATTACK Lazy Player
DUMP








