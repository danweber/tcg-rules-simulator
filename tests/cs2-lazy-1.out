FIELD1:Victim,CS1-02
FIELD2:
GAME:P2 T1 MAIN 9
ECHO: -- play, get suspended immediately
FIELD1:Victim,CS1-02
FIELD2:Lazy,CS2-06,REST
GAME:P2 T1 MAIN 4
ECHO: -- try to attack, fail
FIELD1:Victim,CS1-02
FIELD2:Lazy,CS2-06,REST
GAME:P2 T1 MAIN 4
